src/quo2/components/text_combinations/view.cljs

```diff
@@ -0,0 +1,51 @@
+(ns quo2.components.text-combinations.view
+  (:require
+    [quo2.theme :as theme]
+    [quo2.components.markdown.text :as text]
+    [quo2.components.text-combinations.style :as style]
+    [react-native.core :as rn]))
+
+(defn icon
+  [source size]
+  [rn/image
+   {:source (if (string? source)
+              {:uri source}
+              source)
+    :style  {:border-radius 50
+             :border-width  0
+             :border-color  :transparent
+             :width         size
+             :height        size}}])
+
+(defn view-internal
+  [{:keys [container-style
```

Comment:
In Figma there's a boolean property named `Emoji Dash`, is it being ignored on purpose?